### DISCUSSION TITLE
Can InstancedRigidBodies have their own Attractors?

### DISCUSSION THREAD

**Discussion by casualWaist** [pmndrs/react-three-rapier#365] [Q&A] [answered]
When I nest an Attractor in an InstancedRigidBodies tag I just get a single attraction point at the center of the scene despite all of the instances being placed properly in the scene. Not sure if I have multiple attractors at the center or it's just one. It works with a single RigidBody and I'm not sure if I'm reaching for a capability beyond what was intended.

**Comment by wiledal** [comment, marked as answer]
Never tried this! But it might work to add an attractor within the `colliderNodes` array, as those are directly injected into the RigidBodies as children.

```ts
<InstancedRigidBodies colliderNodes={[
  <Attractor />
]} />
```

Again, I haven't tried this, but it should technically work.

**Reply by casualWaist** [reply]
Oh cool! Thanks! I'll give it a try and let you know how it works.

**Reply by casualWaist** [reply]
So, it kind of works! Attractors do get there initial position from the RigidBodies but, in my case, don't update when the body moves like the nested one in the Attractors demo.
